sql/types/json_value.go

```diff
@@ -156,7 +161,7 @@ func ContainsJSON(a, b interface{}) (interface{}, error) {
 	switch a := a.(type) {
 	case []interface{}:
```

Comment:
might as well use the other alias next to the `JsonObject` additions?

Reply:
Done.